### DISCUSSION TITLE
Is this benchmark still valid?

### DISCUSSION THREAD

**Discussion by muhajirdev** [replit/kaboom#848] [General]
I found this benchmark, which shows that kaboom FPS (3 fps)
https://github.com/Shirajuki/js-game-rendering-benchmark

meanwhile I foundout that kaboom 3000 is much faster https://kaboomjs.com/blog/3000-perf

So, is the benchmark still valid?

**Comment by lajbel** [comment]
No, is not valid anymore, the original benchmark uses 2000 version
